zproject/settings.py

```diff
@@ -826,6 +826,7 @@ def get_secret(key):
             'node_modules/handlebars/dist/handlebars.runtime.js',
             'third/marked/lib/marked.js',
             'generated/emoji/emoji_codes.js',
+            'generated/pygment_data.js',
```

Comment:
I think maybe this should be called `pygments_data.js`, given that `pygments` is the name of the Python library?